Applications/ApplicationsLib/ProjectData.cpp

```diff
@@ -194,8 +194,8 @@ bool ProjectData::isMeshNameUniqueAndProvideUniqueName(std::string& name) const
         if (count > 1)
             cpName = cpName + "-" + std::to_string(count);
 
-        for (auto it = _mesh_vec.begin(); it != _mesh_vec.end(); ++it)
-            if (cpName.compare((*it)->getName()) == 0)
+        for (auto it : _mesh_vec)
```

Comment:
I think now the name `it` is inappropriate, because now it's a mesh pointer, and not a vector iterator anymore.
Maybe there are other occasions of that "misnomer", too.